.gitmodules

```diff
@@ -13,3 +13,15 @@
 [submodule "sonic-swss"]
 	path = src/sonic-swss
 	url = https://github.com/Azure/sonic-swss
+[submodule "src/p4-switch/switch"]
+	path = src/p4-switch/switch
+	url = https://github.com/lguohan/switch
```

Comment:
It is possible to point to Azure organization repo? Pointing to personal repo is weird.